### PR TITLE
CONCF-125 add wikiaVariables caching for 1 minute

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/CONCF-125

### CODE REVIEW

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 1]
```diff
@@ -256,6 +257,9 @@ public function getWikiVariables() {
 
 		$this->response->setVal( 'data', $wikiVariables );
 		$this->response->setFormat( WikiaResponse::FORMAT_JSON );
+
+		//cache wikiaVariables for 1 minute		
+		$this->response->setCacheValidity( self:: WIKIA_VARIABLES_CACHE_TTL);
```
Comment (macbre) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: Can we cache it for a bit longer time, like an hour or a day? And invalidate the cache when required.

Comment (dianafa) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: invalidating cache will require quite a lot of effort in this case. Can I find you tomorrow and can we talk about this?

Comment (macbre) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: Sure :)

**Comment by hakubo** [comment]
LGTM

**Pull request merged**